apps/client/src/features/Events/EventCard.tsx

```diff
@@ -0,0 +1,56 @@
+import { FC } from 'react';
+import { Paper, Button, Stack, Typography, styled } from '@mui/material';
+
+interface EventCardProps {
+  name: string;
+  desc: string;
+  start: string;
+  end: string;
+}
+
+const InfoStack = styled(Stack)(() => ({
+  gap: '1rem',
+  textOverflow: 'ellipsis',
+  overflow: 'hidden',
+}));
+
+const GenericTypography = styled(Typography)(() => ({
```

Comment:
For some reason the title and description are all being set to uppercase text. I'm not sure if this was intentional or a bug but I don't see text-transform: uppercase anywhere so I assume it is a bug. Will still accept PR as this is just a styling thing.